docs/usage/build_fms_model.rst

```diff
@@ -107,7 +98,7 @@ The quickstart instructions can be used with the null model example located in t
   # Create the compile script
   fre make compile-script -y null_model.yaml -p ncrc5.intel23 -t prod -t debug
 
-  # Create and run the compile script
```

Comment:
for baremetal and container steps you only have te --execute example. Do the same here.

Reply:
Also, more specific detail: "Create and run one compile script for each target specified" as it is multi-target steps